style.css

```diff
@@ -47,22 +72,67 @@ h1 {
 
 .menu {
   /*add styles for the menu here*/
+  display:flex;
+  color:white;
+  
+}
+.menu li {
+  list-style-type: none;
+  padding-left: 25px;
+  padding-right:10px;
 }
 
 
 header {
   /*add styles for the header here*/
+  display:flex;
+  justify-content:center;
 }
 
 .headerImg {
+  
   /*add styles for the header image here*/
+  width:250px;
+  margin-bottom:15px;
 }
 
 .social {
   /*add styles for the social icons here*/
+  width:600px;
+  display: flex;
+  flex-direction:row;
+  justify-content:space-evenly;
+  
+
 }
 
 footer {
   /*add styles for the footer here*/
+  display: flex;
+  flex-direction:row;
+  justify-content:center;
+
+}
+
+footer li{
+  /*add styles for the footer here*/
+  
+  list-style-type: none;
+  color:white;
+ 
+}
+
+@media (max-width: 850px) {
+  
```

Comment:
Use mobile first design.  max-width: 850px  will target all devices less that length (which includes cell phones)

What mobile first design means is Default styles should be for mobile. Queries after that then target progressively larger screens. Example below:

`body {
    /* default styles here, 
       targets mobile first */
}

@media screen and (min-width:480px) {
  body {
    /* style changes when the screen gets larger i.e tablets*/
  }
}

@media screen and (min-width:800px) {
  body {
     /* And even larger i.e desktops*/
  }
 }`